src/main/java/org/snomed/snowstorm/core/data/services/BranchReviewService.java

```diff
@@ -567,7 +818,6 @@ Set<Long> createConceptChangeReportOnBranchForTimeRange(String path, Date start,
 		logger.debug("Collecting description changes for change report: branch {} time range {} to {}", path, start, end);
 		AtomicLong descriptions = new AtomicLong();
 		NativeSearchQuery descQuery = newSearchQuery(updatesDuringRange)
-				.withFilter(termQuery(Description.Fields.TYPE_ID, Concepts.FSN))
```

Comment:
I found this one line was all that was required for MAINT-1038, however, it also meant multiple synonym merges were no longer silent. I therefore wrote the `onlyDifferenceIsDescriptionSynonym` method.